components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -148,7 +148,8 @@ const ContributorHighlightCard = ({
     ) {
       const { apiPaths } = generateRepoParts(highlight.highlightLink);
       const { repoName, orgName, issueId } = apiPaths;
-      const repoIcon = getAvatarByUsername(orgName, 60);
+      // default to the GitHub avatar if we can't find the avatar for the organization.
```

Comment:
I can extract this into `getAvatarByOrgName` and default to `"github"` it it's `null`, but it's only in two places at the moment, and I'm also not sure why the org name is not available in these two scenarios.